erase-install.sh

```diff
@@ -273,10 +273,11 @@ dep_notify() {
 }
 
 dep_notify_progress() {
-    # function for DEPNotify to show progress while the installer is being prepared
+    # function for DEPNotify to show progress while the installer is being downloaded or prepared
     last_progress_value=0
     current_progress_value=0
 
+    if [ $1 = startoinstall ]; then
```

Comment:
Can you switch this to startosinstall? I know it doesn't really matter but it bugs me 😀

Reply:
Ah don't worry, I'll do it.